ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/events/TransactionsReader.scala

```diff
@@ -127,9 +129,12 @@ private[dao] final class TransactionsReader(
           dispatcher.executeSql(dbMetrics.getTransactionTrees) { implicit connection =>
             query.asVectorOf(EventsTable.rawTreeEventParser)
           }
-        Timed.future(
-          future = rawEvents.flatMap(Future.traverse(_)(deserializeEntry(verbose))),
-          timer = dbMetrics.getTransactionTrees.translationTimer,
+        rawEvents.flatMap(
+          es =>
```

Comment:
Can you elaborate `es`, please?

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6043/commits/b49c2ae5f520d554734406e597d11350877e28b6

Reply:
No it wasn't. 😛